### PR TITLE
[CURATOR-327] -  NamespaceImpl does not report full error message to caller

### PR DESCRIPTION
**Pull request by rmauge**

### CODE REVIEW

**Comment by Randgalt** [comment]
Is there a Jira issue for this? Please name is PR "[CURATOR-NNN] - Fixes incomplete error message sent to caller"

**Comment by rmauge** [comment]
My apologies, I should have read the commit protocol beforehand. I have created the JIRA issue and updated the title.

File: curator-framework/src/main/java/org/apache/curator/framework/imps/NamespaceImpl.java [hunk 1]
```diff
@@ -45,7 +45,7 @@
             }
             catch ( IllegalArgumentException e )
             {
-                throw new IllegalArgumentException("Invalid namespace: " + namespace);
+                throw new IllegalArgumentException("Invalid namespace: " + namespace + ", " + e.getMessage());
```
Comment (Randgalt) [inline curator-framework/src/main/java/org/apache/curator/framework/imps/NamespaceImpl.java]: getMessage() can return `null`. Is that OK?

Comment (rmauge) [inline curator-framework/src/main/java/org/apache/curator/framework/imps/NamespaceImpl.java]: Hmm, it general yes of course. But validatePath did not have such a path. I can correct this if needed?

Comment (Randgalt) [inline curator-framework/src/main/java/org/apache/curator/framework/imps/NamespaceImpl.java]: It's not a big deal, it would just print "null".

**Pull request closed without merging**